selfdrive/car/toyota/carcontroller.py

```diff
@@ -59,7 +59,7 @@ def update(self, CC, CS, now_nanos):
     self.steer_rate_counter, apply_steer_req = common_fault_avoidance(abs(CS.out.steeringRateDeg) >= MAX_STEER_RATE, lat_active,
                                                                       self.steer_rate_counter, MAX_STEER_RATE_FRAMES)
 
-    if not lat_active:
+    if not (lat_active and apply_steer_req):
```

Comment:
Are you sure? This behavior is the expected one as far as I remember. The line above is for avoiding a fault specific to Toyota that occurs when the steering wheel angle rate is above 100 deg/s for a few frames. The only way to prevent that is to set the steer fault bit to 0 for one frame while maintaining the torque value so we don't have to wind back up

Reply:
Ah got it! since I was testing with sending that same message > 20 times while the tolerance for fault is 18 frames. I see, will close this PR!